crates/settlement-clients/ethereum/src/tests/mod.rs

```diff
@@ -167,7 +168,7 @@ mod settlement_client_tests {
         let nonce = ethereum_settlement_client.get_nonce().await.expect("Unable to fetch nonce");
 
         // keeping 11 elements because the code accesses 10th index as program output
```

Comment:
Better to update the comment and give an explanation as to why the 16th element is accessed now

Reply:
updating the comment, where is 16 accessed?

Reply:
my bad read the code wrong, actually here we are updating the program_output vec size from 11 to 16, it seems to be because of Y_LOW_POINT_OFFSET being 14, but I am lacking the context here, could you provide a liner or 2 ?

Reply:
We shifted to new contract that has latest format and snos also gives the result in that format so we changed the index in order to make the test pass as Y_LOW_POINT_OFFSET and Y_HIGH_POINT_OFFSET are accessed during state update.